pkg/api/dashboard_permission.go

```diff
@@ -99,7 +100,7 @@ func (hs *HTTPServer) UpdateDashboardPermissions(c *models.ReqContext, apiCmd dt
 		return response.Error(403, "Cannot remove own admin permission for a folder", nil)
 	}
 
-	if err := updateDashboardACL(hs, dashID, items); err != nil {
+	if err := updateDashboardACL(context.Background(), hs.SQLStore, dashID, items); err != nil {
```

Comment:
```suggestion
	if err := updateDashboardACL(c.Req.Context(), hs.SQLStore, dashID, items); err != nil {
```

Why are we not passing the request's context here?